### DISCUSSION TITLE
Yubikey/Yubico icon

### DISCUSSION THREAD

**Discussion by optimizasean** [tailwindlabs/heroicons#449] [Ideas]
It would be nice to have a yubikey/yubico icon added for those companies that use them for secure access to resources.  Yubico's Yubikeys are one of the primary hardware security keys across the government and the private sector.  They have many different versions that work across NFC, with iPhones and Androids, FIPS compliant, and more.  Most OS now support them as well as a growing number of services including GitHub and most password managers.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
